x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/knowledge_base/knowledge_base.spec.ts

```diff
@@ -25,6 +25,8 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
   const observabilityAIAssistantAPIClient = getService('observabilityAIAssistantApi');
 
   describe('Knowledge base', function () {
+    // see details: https://github.com/elastic/kibana/issues/207792
+    this.tags(['failsOnMKI']);
```

Comment:
Yesterday and the day before I merged PRs to reduce the flakiness of these tests:

- https://github.com/elastic/kibana/pull/207715
- https://github.com/elastic/kibana/pull/207272

Did you notice flakiness after this?

Reply:
Honestly the svl tests have been failing since early Monday.
There's been a lot of slack chatter in myriad channels, but we think [this](https://github.com/elastic/kibana/pull/207899) will fix it.

Reply:
🤞